books/RayTracingTheNextWeek.html

```diff
@@ -2273,6 +2273,7 @@
 
   $$ Ax + By + Cz + D = 0 $$
 
+where $A,B,C,D$ are just numbers, and $x,y,z$ are the values of a point $\mathbf{P} = (x,y,z)$.
```

Comment:
Sorry. This was my bad. I think maybe we should change to

where $A,B,C,D$ are just constants, and $x,y,z$ are the values of a point $\mathbf{P} = (x,y,z)$.

Reply:
Done.